doc/modules/model_inspection.rst

```diff
@@ -0,0 +1,48 @@
+.. _model_inspection:
+
+================
+Model Inspection
+================
+
+.. currentmodule:: sklearn.model_inspection
+
+A trained machine learning model can achieved good predictive performance in 
```

Comment:
It may already be appropriate to indicate in this first sentence whether the functions are constrained to supervised machine learning models, or not

Reply:
Several typos here. Alternative text:
"Predictive performance is often the main goal of developing a machine learning model. Yet summarising performance with an evaluation metric is often insufficient: it assumes that the evaluation metric and test dataset perfectly reflect the target domain, which is rarely true. The :mod:`sklearn.model_inspection` model provides tools to help understand the predictions output a model and what affects them. This can be used to evaluate model and evaluation assumptions and biases, design a better (e.g. simpler) model, or to diagnose issues with model performance."

I'm sure a lot more can be said here.